examples/zephyr/fw_update/boards/mimxrt1024_evk.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_GOLIOTH_SAMPLE_DHCP_BIND=y
 CONFIG_NET_L2_ETHERNET=y
 CONFIG_NET_DHCPV4=y
 CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_BOOT_DELAY=1
```

Comment:
This is specified in milliseconds, should it be 1000?

I also wonder if this should be part of the twister configuration instead of the board configuration, as we only need it for testing. On the other hand, this could be a material change in behavior between sample and test that might affect the accuracy of test results.